src/main/java/org/sonatype/gradle/plugins/scan/common/DependenciesFinder.java

```diff
@@ -172,20 +175,48 @@ <T> Stream<T> getDependencies(
       return function.apply(originalConfiguration.getResolvedConfiguration());
     }
     catch (ResolveException e) {
-      Configuration copyConfiguration = createCopyConfiguration(project);
+      Configuration copyConfiguration = copyDependencies(project, originalConfiguration, false);
 
-      originalConfiguration.getAllDependencies().all(dependency -> {
-        if (dependency instanceof ProjectDependency) {
-          Project dependencyProject = ((ProjectDependency) dependency).getDependencyProject();
-          project.evaluationDependsOn(dependencyProject.getPath());
-        }
-        else {
-          copyConfiguration.getDependencies().add(dependency);
+      try {
+        return function.apply(copyConfiguration.getResolvedConfiguration());
+      }
+      catch (Exception lastResortException) {
```

Comment:
Question. Instead of catching large scoped `Exception` is it worth catching the more specific `ResolveException` ?

Reply:
> Question. Instead of catching large scoped `Exception` is it worth catching the more specific `ResolveException` ?

Done: 67073e7b27d33ade3f14e05627408a38e954715d